### PR TITLE
Show user view

### LINKED ISSUE #69: Show User View
Show User View

Image

### PR DESCRIPTION
**Pull request by Zama15**
# Description
- issue #69 

my last PR should include this one but last time i didnt notice about that i modify the posts/index route, now is the user/show, it was just add the follows buttons. some changes were made in the cards since last time though. 
I receive an advice from a mentor  to use the dialog html element for the modal, i improve that part with this PR, the routines view made by me were modified in order to correct the modal with the new feature.
Correct the like button bug were the button do not trigger the action.

## Screenshots

image
image
image

### CODE REVIEW

File: app/views/follower/_follower.html.erb [hunk 1]
```diff
@@ -1,10 +1,14 @@
-<div data-controller="follow">
+<div class="" data-controller="follow">
```
Comment (PudinDeveloper) [inline app/views/follower/_follower.html.erb]: Is this class supposed to be empty? 👀 If there's a specific reason for having an empty class, it's a good practice to document it in comments for clarity. If not, please remove the class of the div :D.
```suggestion
<div data-controller="follow">
```

**Review by PudinDeveloper — CHANGES_REQUESTED** [review]
Make sure to take a look on the requested changes and I'll be happy to approve these changes.

**Review by MauricioNC — APPROVED** [review]

**Review by angel23v — APPROVED** [review]

File: app/views/follower/_follower.html.erb (hunk 1 above)
Comment (Zama15) [reply app/views/follower/_follower.html.erb]: i forgot delete that, but now is done

Comment (PudinDeveloper) [reply app/views/follower/_follower.html.erb]: @Zama15 Cool, I am gonna approve it ☕️

**Review by PudinDeveloper — APPROVED** [review]

**Review by BlazeDex — APPROVED** [review]

**Pull request merged**